Doc/c-api/sys.rst

```diff
@@ -291,19 +291,21 @@ accessible to C code.  They all work with the current interpreter thread's
    Raise an auditing event with any active hooks. Return zero for success
    and non-zero with an exception set on failure.
 
+   The *event* string argument must not be *NULL*.
+
    If any hooks have been added, *format* and other arguments will be used
    to construct a tuple to pass. Apart from ``N``, the same format characters
    as used in :c:func:`Py_BuildValue` are available. If the built value is not
-   a tuple, it will be added into a single-element tuple. (The ``N`` format
-   option consumes a reference, but since there is no way to know whether
-   arguments to this function will be consumed, using it may cause reference
-   leaks.)
```

Comment:
Why remove the explanation? Is it more confusing to know why you shouldn't use `N`? Or better to force people to figure it out themselves?

Reply:
I don't recall. I made many changes in this PR. I restored the previous doc.